patches/nuget-client/0007-build-for-.NET-5.patch

```diff
@@ -0,0 +1,67 @@
+From a158ba6c9bdf7465264a5de8ee62e5d3572b961a Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Thu, 12 Nov 2020 15:51:37 +0100
+Subject: [PATCH] nuget.client: build for .NET 5
+
+---
+ build/common.project.props                              | 4 ++--
+ build/packages.targets                                  | 4 ++--
+ src/NuGet.Core/NuGet.Frameworks/NuGet.Frameworks.csproj | 2 +-
+ 3 files changed, 5 insertions(+), 5 deletions(-)
+
+diff --git a/build/common.project.props b/build/common.project.props
+index 03c93ae65..4679fd57a 100644
+--- a/build/common.project.props
++++ b/build/common.project.props
+@@ -13,11 +13,11 @@
+     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+     <NETFXTargetFrameworkVersion>v4.7.2</NETFXTargetFrameworkVersion>
+     <NETFXTargetFramework>net472</NETFXTargetFramework>
+-    <NETCoreTargetFramework>netcoreapp2.1</NETCoreTargetFramework>
++    <NETCoreTargetFramework>netcoreapp5.0</NETCoreTargetFramework>
```

Comment:
Shoudn't this be `net5.0`? I thought `netcoreapp5.0` is just an obsolete alias for that?

Reply:
The upstream repo hasn't adopted `net5.0` yet, and some of the build logic relies on it being `netcoreapp5.0`.

Reply:
Thanks for clarifying that!